### DISCUSSION TITLE
Hot Reload

### DISCUSSION THREAD

**Discussion by mtbayley** [Megabit/Blazorise#2927] [General]
I can't find this information anywhere...
Does anyone know if Blazor Hot Reload will support libraries such as Blazorise?
I updated to VS 2022 Preview 4 and I get stuff like this happen when I make changes:
![image](https://user-images.githubusercontent.com/34929316/133660540-d180a331-0208-4405-a68c-979ca22f3ee6.png)
It changed my h3 and turned it into a heading!

**Comment by stsrki** [comment]
I still haven't tried Hot Reload. But to me, this seems like a bug in VS.

**Comment by David-Moreira** [comment]
Hello, 
should support any library that is compiled against net6 as far as I am aware, probably with some caveats where you have to hard reload sometimes. (gotta check hot reload docs).  

The problem you are having seems something else. Somehow in your razor file the Heading component is not recognised and compiled as a blazor component and instead as regular HTML. Can you post a print of your razor file?

**Reply by mtbayley** [reply]
![image](https://user-images.githubusercontent.com/34929316/133681721-e826b4f8-ade8-40c3-ac5a-3949a0e8f6e0.png)

**Reply by mtbayley** [reply]
You might be right, maybe I need a version of Blazorise complied against net6

**Reply by stsrki** [reply]
Probably yes. We tried to compile it with the last preview but it didn't work because we are targeting both 5 and 6. Maybe we could try the latest version.

**Reply by mtbayley** [reply]
I'll be patient! Thanks for the answers

**Comment by mtbayley** [comment]
I just wanted to let you guys know that I have tested this again with Blazorise 0.9.4.6 and hot reload works great!

**Reply by stsrki** [reply]
Nice :)
